multimodal/src/autogluon/multimodal/predictor.py

```diff
@@ -677,7 +680,11 @@ def fit(
             self._detection_anno_train = train_data
             train_data = from_coco_or_voc(train_data, "train")
             if tuning_data is not None:
+                self.detection_anno_train = tuning_data
                 tuning_data = from_coco_or_voc(tuning_data, "val")
+                if max_tuning_num is not None:
```

Comment:
Will this work for classification or other problem types as well? Because this argument might solve the problem we had before. That is validation takes too much time, which leads to model fitting doesn't honor user specified time limit.

Reply:
Sure, I'll do it in another PR (combine this with [these line](https://github.com/autogluon/autogluon/blob/fedabac33b8595d07564074411da20251a1c5048/multimodal/src/autogluon/multimodal/predictor.py#L712-L727)).